backend/src/database/seed-certifications.ts

```diff
@@ -0,0 +1,16 @@
+import { datasource } from './datasource';
+import { certs } from './queries';
+
+const seedLocation = async () => {
```

Comment:
Am I missing when these are actually going to run?

Reply:
Yes, they will run in the TC-500-personnel PR. Just trying to split these up for readability without breaking the tests.

Reply:
Last one, this should be `seedBcwsCertifications` and the other seed should be `seedFireCentreLocations`